lib/modules/platform/github/schema.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod';
+
+export const IssueSchema = z.object({
```

Comment:
Will this "gracefully" ignore any issue with e.g. missing state field? I'm not sure if I need to control that behavior in zod

Reply:
No, the parsing will fail. You can use `catch` to provide a default value.

Reply:
The whole list will fail or just the element with missing data?

Reply:
The whole list will fail to parse.

Reply:
Is there an example of how to have mandatory fields like this but then only filter the failing ones instead of the entire result?

Reply:
You can use `.nullable().catch(null)`. If than your validation for a field fails only the field will be set to null.

Reply:
I need the field to be mandatory and then skip any object missing any mandatory fields

Reply:
Then you can use `LooseArray` as described above. https://github.com/renovatebot/renovate/pull/27539/files#diff-5b6656b554800147ca327d192449529a03438fecb74935ab2967cd47d0d2d983R27